themes/ddbasic/template.php

```diff
@@ -1299,3 +1246,50 @@ function ddbasic_libraries_info() {
     ),
   );
 }
+
+/**
+ * Implements hook_metatag_metatags_view_alter().
+ */
+function ddbasic_metatag_metatags_view_alter(&$output, $instance, $options) {
+  if($instance == 'panels:ding_event_node_view') {
```

Comment:
This entire function is heavily tied to the event structure and not very theme specific. Is there a reason why it is not placed in the `ding_event` module?